react/react_testing/mocking_callbacks_and_components.md

```diff
@@ -166,4 +166,4 @@ The following questions are an opportunity to reflect on key topics in this less
 
 This section contains helpful links to related content. It isn't required, so consider it supplemental.
 
-- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+- Dive deeper into [act()](https://codilime.com/blog/why-should-you-be-grateful-for-act-warnings-in-react-tests/) and understand its most common warning and when act() is not nessesary.
```

Comment:
```suggestion
- Dive deeper into [act()](https://codilime.com/blog/why-should-you-be-grateful-for-act-warnings-in-react-tests/) and understand its most common warning and when act() is not necessary.
```

Reply:
@Rllopez0410 Did you mean to hit resolve? Just a little confused seeing the requested fix marked as resolve but no fix pushed. I've unmarked it now

Reply:
Yeah, I thought I clicked on resolve. Sorry about that.